include/flash_map_pm.h

```diff
@@ -51,4 +47,11 @@
 #define FLASH_AREA_LABEL_EXISTS(label) \
 	IS_ENABLED(PM_IS_ENABLED(label))
 
+
+#if (CONFIG_SETTINGS_FCB || CONFIG_SETTINGS_NVS) && CONFIG_FILE_SYSTEM_LITTLEFS
+#if !(FLASH_AREA_LABEL_EXISTS(littlefs_storage))
+#error "littlefs needs it exclusive partition when used along with either FCB or NVS"
```

Comment:
Change the message so that it would indicate expected partition name littlefs_storage, to appear.